cfgov/unprocessed/css/enhancements/notifications.less

```diff
@@ -0,0 +1,7 @@
+/* CF Enhancements */
+.m-notification__subtle {
+    border-width: 0;
+    .m-notification_message {
+	    font-size: 1em;
+	}
```

Comment:
Too far indented.
What does this notification look like? Is it absolutely necessary we have a variation of notifications?

Reply:
Normal:

![screen shot 2017-11-09 at 10 02 12 am](https://user-images.githubusercontent.com/1280430/32615310-15c26dcc-c535-11e7-95a4-db2169c54ead.png)

Subtle:

![screen shot 2017-11-09 at 10 04 25 am](https://user-images.githubusercontent.com/1280430/32615413-655c8e4e-c535-11e7-90bd-68765273dc84.png)

I know there's an ongoing HUB discussion about call out text, but I don't know where that's headed. This design is already live, just living in the FWB app markup.

Reply:
Okay the beta banner should probably use this too. Can you add a TODO at https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/css/molecules/global-banner.less#L13

Reply:
Any changes there should go through that HUB discussion. I've added both of these examples to that issue.